src/main/scala/com/cloudera/sparkts/TimeSeries.scala

```diff
@@ -206,6 +206,16 @@ class TimeSeries[K](val index: DateTimeIndex, val data: DenseMatrix[Double],
     }
   }
 
+  def toSamples(): IndexedSeq[(DateTime, Vector[Double])] =
+  {
+    (0 until data.rows).map(rowIndex => (index.dateTimeAtLoc(rowIndex), data(rowIndex, ::).inner.toVector))
+  }
+
+  def toRowSequence(): IndexedSeq[(Int, Vector[Double])] =
```

Comment:
I'm somewhat unsure we need this method, as it seems like it could be just as easily accomplished with `zipWithIndex`?